website/content/docs/commands/billing/index.mdx

```diff
@@ -0,0 +1,43 @@
+---
+layout: docs
+page_title: billing - Command
+description: |-
+  The "billing" command retrieves the number of active users to help you predict user-based pricing.
+---
+
+# billing
+
+Command: `boundary billing`
+
+The `billing` command retrieves the number of active Boundary users in your organization to help you predict billing information.
+
+An active user is any unique Boundary user resource that has performed at least one authenticated action during the relevant billing period.
+Users perform authenticated actions when they call API endpoints that require a valid auth token.
+Boundary provides auth tokens to users when they authenticate using an auth method.
+
+## Examples
+
+The following command queries the number of active users between September 2023 and February 2024:
+
+```shell-session
+$ boundary billing monthly-active-users -start-time="2023-09" -end-time="2024-02"
```

Comment:
I'd suggest using the multi-line format to make this more readable, since we do this for most other CLI docs

```suggestion
$ boundary billing monthly-active-users /
     -start-time="2023-09" /
     -end-time="2024-02"
```

Reply:
Great catch, thank you!